app/build.gradle

```diff
@@ -30,6 +31,7 @@ dependencies {
     implementation 'com.google.firebase:firebase-core:16.0.6'
     // Cloud Firestore
     implementation 'com.google.firebase:firebase-firestore:18.0.0'
+    implementation 'com.firebaseui:firebase-ui-firestore:4.3.1'
     // Google Play - Location
     implementation 'com.google.android.gms:play-services-location:16.0.0'
     // Firebase Authentication

```

Comment:
Consider collecting all of the Firebase dependencies together (and also placing all of the implementation dependencies above the testImplementation and androidTestImplementation dependencies).